sbom_formatter.go

```diff
@@ -0,0 +1,27 @@
+package libcnb
+
+import "io"
+
+// SBOMFormat represents the mapping of a formatted SBOM content to its file
+// extension on the filesystem.
+type SBOMFormat struct {
+	Extension string
+	Content   io.Reader
+}
+
```

Comment:
```suggestion

//go:generate mockery --name=SBOMFormatter --case=underscore

```

We typically add this comment. That should make it generate the mock.

Reply:
Yeah that is my bad. I just forgot to write the file one second.

Reply:
Looks good, thanks.